WcaOnRails/config/locales/en.yml

```diff
@@ -23,20 +23,28 @@ en:
   common:
     here: "here"
     wca_id: "WCA ID"
-    search: "Search"
     all_regions: "All regions"
+  datepicker:
+    # Do *NOT* change these, they would break fields' validation
+    dformat: "%Y-%m-%d"
+    pformat: "YYYY-MM-DD"
+    dayViewHeaderFormat: "MMMM YYYY"
+  timepicker:
+    # Do *NOT* change these, they would break fields' validation
+    dformat: "%R"
+    pformat: "HH:mm"
```

Comment:
Since when a translation is missing rails uses english I think it's fine to have a single file with the shared (constant) keys!

I did this, the problem is that they are now considered as missing for the `fr` locale, and putting them in a language-scoped `ignore_missing` (like what is done [here](https://github.com/glebm/i18n-tasks/blob/master/spec/fixtures/config/i18n-tasks.yml.erb#L58) for `ignore_eq_base`) just doesn't work: when I remove the keys from the `en` locale it doesn't consider them as missing, which makes me think they are ignored too...

Anyway, it won't break the spec but it's a bit weird when you want to check for missing translation for foreign languages.